.env.development

```diff
@@ -7,7 +7,7 @@ BULK_SEARCH_API_ENABLED=true
 DEFAULT_API_VERSION=2
 DIFFERENCES_TO_EMAILS=Online Trade Tariff Support <hmrc-trade-tariff-support-g@digital.hmrc.gov.uk>
 DUMP_CDS_DATA_AS_JSON=false
-EXCESS_QUERY_THRESHOLD=1000
+EXCESS_QUERY_THRESHOLD=2000
```

Comment:
This was a suggested local variable change - this will change for all developers

Please use `.env.development.local`

Reply:
Done